api/build.gradle

```diff
@@ -86,7 +86,45 @@ configurations {
   }
 }
 
-def swagger2CodegenDir = "$buildDir/swagger2"
+// Shared variables for Swagger codegen. Use "ext" rather than "def" to allow
+// access globally, including from within helper functions.
+ext.swaggerCodegenBasePkg = 'org.pmiops.workbench'
+ext.swagger2CodegenDir = "$buildDir/swagger2"
+// This path is the Java sources root for finding any overridden Swagger-codegen
+// templates we may be using.
+ext.swaggerTemplateDir = "$projectDir/src/main/resources/swaggercodegen"
+// Raw options passed to the Swagger codegen CLI.
+ext.swagger2RawOpts = [
+  // By default, the ignore file within the output directory is used. Override
+  // as we don't track the output directory in source.
+  '--ignore-file-override', "$projectDir/.swagger2-codegen-ignore"
+]
+
+// Helper function to define a Java client Swagger codegen target for the Gradle
+// Swagger Generator Plugin: https://github.com/int128/gradle-swagger-generator-plugin#code-generation
+def swagger2JavaClient(yaml, pkg, rawOptExtras = []) {
+  return {
+    inputFile = file("src/main/resources/${yaml}")
+    code {
+      configuration = configurations.__swaggerCodegenV2__
+      language = 'java'
+      outputDir = file(swagger2CodegenDir)
+      templateDir = file(swaggerTemplateDir)
+      // Since the output directory is shared among these swagger tasks, wiping
+      // the directory would result in task invalidation / churn.
+      wipeOutputDir = false
+      rawOptions = swagger2RawOpts + rawOptExtras
+      additionalProperties = [
+        'library'          : 'okhttp-gson',
+        'serializableModel': 'true',
+        'dateLibrary'      : 'java8',
```

Comment:
Is this a change? I  thought  it was defaulting to `joda` today for datetimes, at least in  our own  API models. `java8` is the recommended  replacement for joda, but there would be a bit of fixup  I'd think.

Reply:
It's not a change, see the previous value above: https://github.com/all-of-us/workbench/pull/3907/files#diff-dd8fd6d45c4e2eccf0f278e121d714d8L150